circleci: failed tests (14804): TestDockerJava

The following test appears to have failed:

[#14804](https://circleci.com/gh/cockroachdb/cockroach/14804):

```
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:386)
	at main.main(main.java:58)
I160318 17:29:43.195477 acceptance/cluster/localcluster.go:634  stopping
I160318 17:29:43.195723 acceptance/cluster/localcluster.go:532  event stream done, resetting...: net/http: request canceled
I160318 17:29:43.195743 acceptance/cluster/localcluster.go:555  events monitor exits
--- FAIL: TestDockerJava (18.45s)
	util_test.go:259: expected success: non-zero exit code: 1
=== RUN   TestDockerPHP
I160318 17:29:45.479334 acceptance/cluster/localcluster.go:274  Initializing Cluster AdHoc 3x1:
{"name":"AdHoc 3x1","nodes":[{"count":3,"stores":[{"count":1,"max_ranges":0}]}],"duration":5000000000}
I160318 17:29:45.479538 acceptance/cluster/docker.go:73  ImagePull cockroachdb/builder:20160305-182433 starting
..................
I160318 17:29:45.707693 acceptance/cluster/docker.go:89  ImagePull cockroachdb/builder:20160305-182433 complete
I160318 17:29:46.277479 acceptance/cluster/localcluster.go:566  creating certs (1024bit) in: /home/ubuntu/cockroach/.localcluster.certs.081252519
I160318 17:29:46.384581 acceptance/cluster/localcluster.go:517  events monitor starts
I160318 17:29:46.652897 acceptance/cluster/localcluster.go:465  *** started roach0 ***
--
  trace:     https://127.0.0.1:32854/debug/requests
  logs:      /tmp/circle-artifacts.ELLfbau/acceptance/TestDockerPHP/roach2/cockroach.INFO
  pprof:     docker exec -it 2970c /bin/bash -c 'go tool pprof /cockroach <(wget --no-check-certificate -qO- https://$(hostname):8080/debug/pprof/heap)'
  cockroach: [start --ca-cert=/certs/ca.crt --cert=/certs/node.crt --key=/certs/node.key --host=roach2 --alsologtostderr=INFO --store=path=/data3.0 --join=roach0:26257 --log-dir=/logs/roach2 --logtostderr=false]
I160318 17:29:55.734348 acceptance/replication_test.go:50  waiting for first range to have 3 replicas
panic: test timed out after 5m0s

goroutine 1308 [running]:
panic(0x11afbe0, 0xc820406940)
	/usr/local/go/src/runtime/panic.go:464 +0x3e6
testing.startAlarm.func1()
	/usr/local/go/src/testing/testing.go:725 +0x14b
created by time.goFunc
	/usr/local/go/src/time/sleep.go:129 +0x3a

goroutine 1 [chan receive]:
testing.RunTests(0x178d368, 0x1e189a0, 0xf, 0xf, 0x8b25668cd134e100)
	/usr/local/go/src/testing/testing.go:583 +0x8d2
testing.(*M).Run(0xc8204c9f08, 0x178b300)
	/usr/local/go/src/testing/testing.go:515 +0x81
github.com/cockroachdb/cockroach/acceptance.TestMain(0xc8204c9f08)
	/go/src/github.com/cockroachdb/cockroach/acceptance/main_test.go:48 +0x3e
main.main()
	github.com/cockroachdb/cockroach/acceptance/_test/_testmain.go:80 +0x114

goroutine 17 [syscall, 4 minutes, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1998 +0x1

goroutine 19 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x20ebae0)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1003 +0x64
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:604 +0x8a

goroutine 8 [select, 4 minutes, locked to thread]:
runtime.gopark(0x178dba0, 0xc820022f28, 0x14c9a08, 0x6, 0x18, 0x2)
	/usr/local/go/src/runtime/proc.go:262 +0x163
runtime.selectgoImpl(0xc820022f28, 0x0, 0x18)
	/usr/local/go/src/runtime/select.go:392 +0xa67
runtime.selectgo(0xc820022f28)
	/usr/local/go/src/runtime/select.go:215 +0x12
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal1_unix.go:279 +0x358
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1998 +0x1

goroutine 21 [syscall, 4 minutes]:
os/signal.signal_recv(0x0)
	/usr/local/go/src/runtime/sigqueue.go:116 +0x132
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 22 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/acceptance.TestMain.func1()
	/go/src/github.com/cockroachdb/cockroach/acceptance/main_test.go:39 +0xd8
created by github.com/cockroachdb/cockroach/acceptance.TestMain
	/go/src/github.com/cockroachdb/cockroach/acceptance/main_test.go:47 +0x30

goroutine 1175 [select]:
github.com/cockroachdb/cockroach/acceptance.checkRangeReplication.func1(0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/acceptance/replication_test.go:53 +0x4b8
github.com/cockroachdb/cockroach/util.RetryForDuration(0x37e11d600, 0xc8205e5220, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/util/testing.go:143 +0x180
github.com/cockroachdb/cockroach/util.SucceedsSoonDepth(0x1, 0x7f5419f3b158, 0xc82019e510, 0xc8205e5220)
	/go/src/github.com/cockroachdb/cockroach/util/testing.go:130 +0x3f
github.com/cockroachdb/cockroach/util.SucceedsSoon(0x7f5419f3b158, 0xc82019e510, 0xc8205e5220)
	/go/src/github.com/cockroachdb/cockroach/util/testing.go:124 +0x3e
github.com/cockroachdb/cockroach/acceptance.checkRangeReplication(0xc82019e510, 0x7f5419f03440, 0xc82041e1a0, 0x4a817c800)
	/go/src/github.com/cockroachdb/cockroach/acceptance/replication_test.go:72 +0x226
--
testing.tRunner(0xc82019e510, 0x1e18a48)
	/usr/local/go/src/testing/testing.go:473 +0x98
created by testing.RunTests
	/usr/local/go/src/testing/testing.go:582 +0x892

goroutine 1314 [select]:
google.golang.org/grpc.(*Conn).transportMonitor(0xc820486620)
	/go/src/google.golang.org/grpc/clientconn.go:511 +0x1d3
google.golang.org/grpc.NewConn.func1(0xc820486620)
	/go/src/google.golang.org/grpc/clientconn.go:322 +0x1b5
created by google.golang.org/grpc.NewConn
	/go/src/google.golang.org/grpc/clientconn.go:323 +0x509

goroutine 1074 [select]:
net/http.(*persistConn).writeLoop(0xc8200c6000)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 26 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f5419efceb0, 0x72, 0xc82029c000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc82024ec30, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc82024ec30, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc82024ebd0, 0xc82029c000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82011a350, 0xc82029c000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201fbe10)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 27 [select, 4 minutes]:
net/http.(*persistConn).writeLoop(0xc8201fbe10)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1287 [select]:
net/http.(*persistConn).writeLoop(0xc8201fa340)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1262 [select]:
net/http.(*persistConn).writeLoop(0xc82041e270)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 540 [select]:
net/http.(*persistConn).writeLoop(0xc8200c61a0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 514 [chan receive, 1 minutes]:
database/sql.(*DB).connectionOpener(0xc8201e8370)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 135 [IO wait, 3 minutes]:
net.runtime_pollWait(0x7f5419efcbb0, 0x72, 0xc8204a0000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc820414bc0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820414bc0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820414b60, 0xc8204a0000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82011a1c8, 0xc8204a0000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8200c6410)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 967 [IO wait]:
net.runtime_pollWait(0x7f5419f061b0, 0x72, 0xc82046b000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8205501b0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8205501b0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820550150, 0xc82046b000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a038, 0xc82046b000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8200c7110)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 610 [IO wait]:
net.runtime_pollWait(0x7f5419f060f0, 0x72, 0xc820579000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc820582140, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820582140, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8205820e0, 0xc820579000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82011a1e0, 0xc820579000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8200c6dd0)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 1306 [select]:
google.golang.org/grpc/transport.(*http2Client).controller(0xc8201cc870)
	/go/src/google.golang.org/grpc/transport/http2_client.go:831 +0x5da
created by google.golang.org/grpc/transport.newHTTP2Client
	/go/src/google.golang.org/grpc/transport/http2_client.go:194 +0x153b

goroutine 504 [chan receive, 1 minutes]:
database/sql.(*DB).connectionOpener(0xc820171ce0)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 136 [select, 3 minutes]:
net/http.(*persistConn).writeLoop(0xc8200c6410)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 611 [select]:
net/http.(*persistConn).writeLoop(0xc8200c6dd0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 382 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc8201e8840)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 234 [IO wait, 3 minutes]:
net.runtime_pollWait(0x7f5419efcc70, 0x72, 0xc820461000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc82024e3e0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc82024e3e0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc82024e380, 0xc820461000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82011a1b0, 0xc820461000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201fa680)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 235 [select, 3 minutes]:
net/http.(*persistConn).writeLoop(0xc8201fa680)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1178 [IO wait]:
net.runtime_pollWait(0x7f5419efc430, 0x72, 0xc820599000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc82039ca70, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc82039ca70, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc82039ca10, 0xc820599000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a360, 0xc820599000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8200c7ad0)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 368 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc820524370)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 418 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc820524160)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 1288 [select]:
github.com/docker/engine-api/client/transport/cancellable.Do.func3(0x7f5419f3acd0, 0xc820378280, 0xc820412250, 0xc8205164e0)
	/go/src/github.com/docker/engine-api/client/transport/cancellable/cancellable.go:78 +0xef
created by github.com/docker/engine-api/client/transport/cancellable.Do
	/go/src/github.com/docker/engine-api/client/transport/cancellable/cancellable.go:84 +0x336

goroutine 1315 [select]:
github.com/cockroachdb/cockroach/rpc.(*Context).runHeartbeat(0xc82047eea0, 0xc820048140, 0xc8203cb540, 0xf, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:210 +0x49d
github.com/cockroachdb/cockroach/rpc.(*Context).GRPCDial.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:163 +0x66
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc82039ca80, 0xc8204cc7b0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62

goroutine 339 [IO wait, 1 minutes]:
net.runtime_pollWait(0x7f5419efcd30, 0x72, 0xc820124000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc820397090, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820397090, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820397030, 0xc820124000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82011a228, 0xc820124000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8200c68f0)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 340 [select, 1 minutes]:
net/http.(*persistConn).writeLoop(0xc8200c68f0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 388 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc820524790)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 379 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc820171d90)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 539 [IO wait]:
net.runtime_pollWait(0x7f5419efc070, 0x72, 0xc82057e000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203968b0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203968b0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820396850, 0xc82057e000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a028, 0xc82057e000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8200c61a0)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 968 [select]:
net/http.(*persistConn).writeLoop(0xc8200c7110)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 461 [select]:
net/http.(*persistConn).writeLoop(0xc8201fa9c0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 406 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc8201e8b00)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 460 [IO wait]:
net.runtime_pollWait(0x7f5419efcdf0, 0x72, 0xc82057c000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc820396290, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820396290, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820396230, 0xc82057c000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82011a1d0, 0xc82057c000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201fa9c0)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 1057 [IO wait]:
net.runtime_pollWait(0x7f5419efc2b0, 0x72, 0xc820286000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc82039c1b0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc82039c1b0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc82039c150, 0xc820286000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a2c8, 0xc820286000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8200c6000)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 750 [IO wait]:
net.runtime_pollWait(0x7f5419efc5b0, 0x72, 0xc8205b3000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8205823e0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8205823e0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820582380, 0xc8205b3000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82011a200, 0xc8205b3000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201fba00)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 1277 [IO wait]:
net.runtime_pollWait(0x7f5419efc370, 0x72, 0xc820500000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8200c01b0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8200c01b0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8200c0150, 0xc820500000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82011a250, 0xc820500000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
github.com/cockroachdb/cockroach/acceptance/cluster.(*LocalCluster).monitor(0xc82041e1a0)
	/go/src/github.com/cockroachdb/cockroach/acceptance/cluster/localcluster.go:553 +0xd1
created by github.com/cockroachdb/cockroach/acceptance/cluster.(*LocalCluster).Start
	/go/src/github.com/cockroachdb/cockroach/acceptance/cluster/localcluster.go:577 +0x731

goroutine 751 [select]:
net/http.(*persistConn).writeLoop(0xc8201fba00)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1304 [chan receive]:
github.com/cockroachdb/cockroach/rpc.NewContext.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:99 +0x57
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc82039ca80, 0xc8204a9d80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62

goroutine 1179 [select]:
net/http.(*persistConn).writeLoop(0xc8200c7ad0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1307 [IO wait]:
net.runtime_pollWait(0x7f5419efc7f0, 0x72, 0xc8204a6c00)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8205820d0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8205820d0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820582070, 0xc8204a6c00, 0x400, 0x400, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82011a420, 0xc8204a6c00, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
google.golang.org/grpc/transport.(*http2Client).reader(0xc8201cc870)
	/go/src/google.golang.org/grpc/transport/http2_client.go:754 +0xef
created by google.golang.org/grpc/transport.newHTTP2Client
	/go/src/google.golang.org/grpc/transport/http2_client.go:200 +0x159a

goroutine 1286 [select]:
net/http.(*persistConn).readLoop(0xc8201fa340)
	/usr/local/go/src/net/http/transport.go:1178 +0xd52
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 1261 [IO wait]:
net.runtime_pollWait(0x7f5419efc4f0, 0x72, 0xc82044a000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc82039c140, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc82039c140, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc82039c0e0, 0xc82044a000, 0x1000, 0x1000, 0x0, 0x7f5419f2f050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82011a1f0, 0xc82044a000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4

```
Please assign, take a look and update the issue accordingly.

--- a/sql/pgwire/v3.go
+++ b/sql/pgwire/v3.go
@@ -353,7 +353,6 @@ func (c *v3Conn) handleParse(buf *readBuffer) error {
 		portalNames: make(map[string]struct{}),
 		columns:     cols,
 	}
-	copy(pq.inTypes, inTypeHints)
 	for k, v := range args {
 		i, err := strconv.Atoi(k)
 		if err != nil {
@@ -367,6 +366,9 @@ func (c *v3Conn) handleParse(buf *readBuffer) error {
 		// Grow pq.inTypes to be at least as large as i.
 		for j := len(pq.inTypes); j <= i; j++ {
 			pq.inTypes = append(pq.inTypes, 0)
+			if j < len(inTypeHints) {
+				pq.inTypes[j] = inTypeHints[j]
+			}
 		}
 		// OID to Datum is not a 1-1 mapping (for example, int4 and int8 both map
 		// to DummyInt), so we need to maintain the types sent by the client.

